app/src/main/java/com/toolslab/reviewme/MainActivity.kt

```diff
@@ -2,11 +2,18 @@ package com.toolslab.reviewme
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        Log.d("LOG", "message "+6)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/ReviewMe/blob/f5bf5ff259181e85dc1f7011c492a3b19f1e4e66/app/src/main/java/com/toolslab/reviewme/MainActivity.kt#L13"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>